service/src/main/java/uk/gov/hmcts/reform/fpl/model/Placement.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@Builder(toBuilder = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Placement {
+
+    @JsonProperty("placementChildId")
+    private UUID childId;
+
+    @JsonProperty("placementChildName")
+    private String childName;
+
+    @JsonProperty("placementApplication")
+    public DocumentReference application;
+
+    @JsonProperty("placementSupportingDocuments")
+    private List<Element<PlacementSupportingDocument>> supportingDocuments;
+
+    @JsonProperty("placementOrderAndNotices")
+    private List<Element<PlacementOrderAndNotices>> orderAndNotices;
+
+    @JsonIgnore
+    public Placement setChild(Element<Child> child) {
```

Comment:
That might be an overkill here, but how about going for:
```
public class PlacementBuilder {
    public PlacementBuilder child(Element<Child> child) {
        this.childId = child.getId();
        this.childName = child.getValue().getParty().getFullName();
        return this;
    }
}
```
That way we wouldn't have to change state of `Placement` instance and will return new one instead with `.toBuilder().child().build()`. Lombok just fills the rest of the builder. Thoughts?

Reply:
i think that it is a bit overkill here, to create new class to just set one value.